scheduler/scheduler.go

```diff
@@ -366,9 +395,14 @@ func containerCapacity(numCurrentParents, numCurrentContainers, numContainersToS
 // createParents creates host intent documents for each parent
 func createParents(parent distro.Distro, numNewParents int, pool *evergreen.ContainerPool) []host.Host {
 	hostsSpawned := make([]host.Host, numNewParents)
-
+	options, err := generateParentHostOptions(parent.ProviderSettings, pool)
+	grip.DebugWhen(err != nil, message.Fields{
```

Comment:
In general errors returned from functions like this should be logged at an error level, unless we're expecting the error to happen (in which case it shouldn't be logged)